ee_tests/src/support/core.ts

```diff
@@ -253,6 +254,7 @@ export async function sleep(ms: number) {
  * Ref: http://blog.ng-book.com/taking-screenshots-with-protractor/
  */
 export async function writeScreenshot(filename: string) {
+  mkdirp(dirname(filename));
```

Comment:
This is the most problematic part... this function should not be used directly but unfortunately it is. Since the `mkdirp` is a sync function, it might slow down the execution of tests but IMHO it won't be a measurable effect.